podman/api_connection.py

```diff
@@ -0,0 +1,108 @@
+""" Provides a Connection to a Podman service. """
+import socket
+import urllib.parse
```

Comment:
Am I miss-remembering, or isn't `requests` much better to use?

Reply:
It was not obvious to me how to do ssh tunneling etc with `requests`.  Seems too high level.